src/client/css/variables.css

```
@@ -29,6 +29,8 @@
   --blue-60: #0250bb;
   --blue-90: #09204d;
   --purple-5: #e7dfff;
+  --purple-20: #cb9eff;
+    --purple-60: #7542e5;
```

Suggestion: Identation

```suggestion
  --purple-60: #7542e5;
```

---

Oh sorry, forgot to call this out - I cannot get VSCode to _not_ add this indentation back. I expect it's something in our shared VSCode settings?

---

Actually, looks like it will let me now: af46b139